utils/zerovec/src/flexzerovec/slice.rs

```diff
@@ -23,6 +22,13 @@ pub struct FlexZeroSlice {
     data: [u8],
 }
 
+impl PartialEq for FlexZeroSlice {
+    fn eq(&self, other: &Self) -> bool {
+        self.width == other.width && self.data == other.data
+    }
+}
+impl Eq for FlexZeroSlice {}
```

Comment:
Nit (optional): I usually like a comment explaining why `Eq` is safe to implement... but I guess it's fairly obvious here